packages/aws-cdk-lib/aws-rds/lib/cluster-ref.ts

```diff
@@ -50,6 +50,11 @@ export interface IDatabaseCluster extends IResource, ec2.IConnectable, secretsma
    */
   readonly engine?: IClusterEngine;
 
+  /**
+   * The ARN of the clusteDatabaseClusterr
```

Comment:
Nit: looks like the spelling got a little messed up here -- how about just "The ARN of the database cluster"?

Reply:
sorry, don't know how that got there :sweat_smile: